addons/mail/static/src/core_ui/message_reaction_menu.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionMenu" owl="1">
+        <div class="d-flex modal justify-content-center w-100 h-100" t-attf-class="{{ env.isSmall ? 'align-items-end' : 'align-items-center' }}"
+        t-on-click.stop="() => this.props.close()" 
+        t-on-keydown="onKeydown">
+            <div t-on-click.stop="" class="d-flex" t-attf-class="{{ env.isSmall ? 'w-100 h-50' : 'o-mail-MessageReactionMenu-container' }}">
+                <div class="text-center w-25 pt-2 overflow-auto d-flex flex-column bg-200">
+                    <t t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content">
+                        <button class="btn p-1 rounded-2 mx-3" t-att-class="{ 'bg-300 border-primary': reaction===state.reaction }" t-att-title="reaction.content" t-on-click="() => state.reaction = reaction">
```

Comment:
- `reaction.content` is redundant with span being displayed. Use `shortcode` instead.
- spacing around operator `===`